packages/nimble-components/.storybook/manager.js

```diff
@@ -7,9 +7,10 @@ addons.setConfig({
         filters: {
             patterns: item => {
```

Comment:
I spent some time scanning docs for how to remove this but haven't found anything yet. I'll keep looking but since this is pre-existing (and I'm OOO soon) I'll propose not blocking the PR on it.

Reply:
Sure, we could also skip tracking it until someone notices and brings it up. Seems minor